### PR TITLE
fix: change header color to match new OFF design

### LINKED ISSUE #273: Harmonizing the header color of Hunger Games with the new design
### What

- [ ] Harmonizing the header color of Hunger Games with the new design
### Screenshot

image

### Part of
- #37

### PR DESCRIPTION
**Pull request by abhilipsasahoo03**
### What
Changed header color to harmonize it with new Open Food Facts design

### Screenshot

Screenshot_2022-11-28-00-56-54-024_com android chrome

### Fixes bug(s)
Fixes #273

### CODE REVIEW

**Comment by abhilipsasahoo03** [comment]
@alexfauquette kindly review!

File: src/App.js [hunk 1]
```diff
@@ -139,7 +139,7 @@ export default function App() {
         contrastText: "#ffffff",
       },
       secondary: {
-        main: "#ffefb7",
+        main: "#f6f3f0",
```
Comment (alexfauquette) [inline src/App.js:142]: not sure
```suggestion
        main: "#f2e9e4",
```

**Review by alexfauquette — COMMENTED** [review]
@teolemon

File: src/App.js (hunk 1 above)
Comment (abhilipsasahoo03) [reply src/App.js:142]: @alexfauquette actually I first took screenshot of the website and applied colour to hex code converter to check the colour of top bar to be #f6f3f0. For further confirmation, if this colour has been used, I went through openfoodfacts-server repo and got this:



The `$latte` variable is used which is #f6f3f0

**Comment by abhilipsasahoo03** [comment]
Hi @alexfauquette I've replied to your comment, kindly review :)

**Comment by alexfauquette** [comment]
Maybe it's the correct secondary color, but the top bar should have `#f2e9e4` because of

```scss
// additional top bar color between latte and cappucino
$cafecreme: #f2e9e4;
```

I don't know what should be be the correct colors, so I let @teolemon have a look

**Comment by abhilipsasahoo03** [comment]
> Maybe it's the correct secondary color, but the top bar should have `#f2e9e4` because of
> 
> ```scss
> // additional top bar color between latte and cappucino
> $cafecreme: #f2e9e4;
> ```
> 
> I don't know what should be the correct colors, so I let @teolemon have a look
> 
> I'm moving the PR to `dev` branch to trigger netlify deploy

Okay no issues @alexfauquette 👍

**Comment by abhilipsasahoo03** [comment]
Hey @alexfauquette @teolemon kindly review and let me know if any change is required?

**Comment by VaiTon** [comment]
Hey @abhilipsasahoo03! I see that there are a lot of changes not directly related to this fix. If that's the case, please split this PR into two and let's analyze them one by one! Thank you for your work!

**Comment by abhilipsasahoo03** [comment]
> Hey @abhilipsasahoo03! I see that there are a lot of changes not directly related to this fix. If that's the case, please split this PR into two and let's analyze them one by one! Thank you for your work!

Hi @VaiTon actually all these changes have not been made by me. They are reflected because the base branch has been pushed from master to dev.

**Comment by alexfauquette** [comment]
Yes, and I don't know why it did not trigger the Netlify rendering

By the way, I took some time to check the design resources.

My main concern is the topBar which is now nearly the same color as the background

image

So the minimal to merge the PR would be to add `"#f2e9e4"` as a background to the AppBar (defined in `src/components/ResponsiveAppBar.jsx`)

-----

If you want to go further with the setup of the palette, here is the design system

https://www.figma.com/file/Qg9URUyrjHgYmnDHXRsTTB/New-website-design-(2022)-(Quentin)?node-id=0%3A2&t=2kXx3sHDtOs1vdcH-0

And here is the docs of MUI platte

https://mui.com/material-ui/customization/palette/

**Comment by abhilipsasahoo03** [comment]
> Yes, and I don't know why it did not trigger the Netlify rendering
> 
> By the way, I took some time to check the design resources.
> 
> My main concern is the topBar which is now nearly the same color as the background
> 
> image
> 
> So the minimal to merge the PR would be to add `"#f2e9e4"` as a background to the AppBar (defined in `src/components/ResponsiveAppBar.jsx`)
> 
> -----
> 
> If you want to go further with the setup of the palette, here is the design system
> 
> https://www.figma.com/file/Qg9URUyrjHgYmnDHXRsTTB/New-website-design-(2022)-(Quentin)?node-id=0%3A2&t=2kXx3sHDtOs1vdcH-0
> 
> And here is the docs of MUI platte
> 
> https://mui.com/material-ui/customization/palette/

Hey @alexfauquette thank you so much for the resources! I'll definitely change it to `#f2e9e4`. Btw, do these commits raise a concern? Should I create a new PR instead? Or is it fine to merge it just how it is?

**Comment by abhilipsasahoo03** [comment]
@alexfauquette I've made the change. Kindly check out!

**Review by alexfauquette — APPROVED** [review]
I though you would keep the previous one as a secondary color and only give a special background to the APp bar, but sounds good too

**Pull request merged**